### PR TITLE
Feature/translation for all pages

### PR DESCRIPTION
**Pull request by Avdunusinghe**

### CODE REVIEW

File: src/locales/en-US/translation.json [hunk 1]
```diff
@@ -16,6 +16,49 @@
...
+			"pending": "Pending",
+			"interview": "Interview",
+			"selected": "Selected",
+			"rejected": "Rejected"
+		},
+
+		"table-type-header-label": {
+			"event-table": {
+				"past-event": "Past Event",
+				"upcomming-event": "Upcomming Event"
```
Comment (KaveenLaksitha) [inline src/locales/en-US/translation.json:151]: Check the spelling

File: src/locales/en-US/translation.json [hunk 2]
```diff
@@ -129,7 +132,21 @@
 					"message": "Are you sure about recovering this deleted Webinar?"
 				},
 				"webinar-input-descrption": {
-					"tangs": "Sperate tag names using , (example: ITP, GitHub, Microservice)"
+					"tags": "Sperate tag names using , (example: ITP, GitHub, Microservice)"
```
Comment (KaveenLaksitha) [inline src/locales/en-US/translation.json:190]: Check spelling

**Review by KaveenLaksitha — CHANGES_REQUESTED** [review]
Added some comments. Please take a look.

File: src/locales/en-US/translation.json (hunk 1 above)
Comment (Avdunusinghe) [reply src/locales/en-US/translation.json]: It Fixed

**Pull request merged**